.ko.yaml

```diff
@@ -1,4 +1,4 @@
-defaultBaseImage: gcr.io/distroless/base-debian10
+defaultBaseImage: gcr.io/distroless/base-debian12
 builds:
 - main: .
   env:
```

Comment:
@loosebazooka If we change the default to be CGO_ENABLED=0, can we switch to a different image? CGO is only needed for the pkcs11 library, which we don't use in the production CA.

Reply:
maybe we can change that to 0 and run on ci, last time i think that was not building properly

Reply:
kk testing it out. also switched from base -> static

Reply:
Looks good!